src/python/ProcInfo.py

```diff
@@ -286,9 +286,9 @@ def diffWithOverflowCheck(this, new, old):
 		if new >= old:
 			return new - old;
 		else:
-			max = (1L << 31) * 2;  # 32 bits
+			max = (1 << 31) * 2;  # 32 bits
 			if old >= max:
-				max = (1L << 63) * 2;  # 64 bits
+				max = (1 << 63) * 2;  # 64 bits
```

Comment:
this ProcInfo.py is part of the "send message to MonAlisa" code
which we got from Julia at the dawn of times and have been
moving aorund since w/o touching nor understanding.
Let's ask them for an uptodate, futurized, one.

Note at the top of the file:
Copyright (C) 2006 California Institute of Technology